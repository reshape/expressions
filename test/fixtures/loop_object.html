<p>x</p>
<each loop='value, key in items'>
  <p>{{key}}: {{value}}</p>
</each>
<p>x</p>
