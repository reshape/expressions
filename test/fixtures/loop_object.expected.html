<p>x</p>

  <p>a: b</p>

  <p>c: d</p>

<p>x</p>
